recipes/symengine/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "0.1.0.dev0" %}
+{% set commit = "0086007c3f588e77c9b8ce8e071fee686ffc6b72" %}
+
+package:
+  name: symengine
+  version: {{ version }}
+
+source:
+  fn: {{ commit }}.tar.gz
+  url: https://github.com/symengine/symengine/archive/{{ commit }}.tar.gz
+  sha256: 98a7f11e6ed5796394910cf118dca9f7a812caaac4ddb6b990ef84db9b8acbe2
+
+build:
+  number: 0
+  skip: true        # [win and not py35]
+  features:
+    - vc14          # [win and py35]
+
+requirements:
+  build:
+    - toolchain     # [unix]
+    - cmake
+    - gmp           # [unix]
+    - mpfr          # [unix]
+    - mpc           # [unix]
+    - mpir          # [win]
+    - python        # [win]
+  run:
+    - gmp           # [unix]
+    - mpfr          # [unix]
+    - mpc           # [unix]
+    - mpir          # [win]
+    - python        # [win]
```

Comment:
I'm a little confused. I don't want to use `vc 14` right now. What should I do in this case?